test/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const SPECS_PATH = "./.tmp/specs";
+export const DEFAULT_SPEC_BRANCH = "49fc16354df7211f8392c56884a3437138317d1f";
```

Comment:
What made you decide to use this as the branch? Any reason not to choose `master` as the default?

Reply:
The problem with using master is that every run may potentially produce a different output, and we'll have a hard time maintaining CI and keeping it healthy. By locking to a point in time we won't be exposed to CI breaks for random changes, for example if we target master and a swagger updates its description for an Operation we'll end up with a broken CI.

I took this commit arbitrarily, I think it was the latest commit to master when I started adding smoke tests.

Reply:
Ah I see. That makes sense since the goal here is to make sure autorest.typescript doesn't regress, not that it supports the latest version of a model. Maybe someday we can have a bot that creates a PR with a new branch commit id for a spec when one of the models changes (so we can review/test it).